lib/tarantool_server.py

```diff
@@ -638,7 +637,7 @@ def crash_grep(self):
                     break
                 if assert_fail_re.match(line):
                     pos = len(lines) - rpos
-                    assert_lines = lines[max(0, pos - print_log_lines):pos]
+                    assert_lines = lines[:pos]
```

Comment:
Don’t get why we need to perform any assert search if we going to show almost whole log file.